.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -24,7 +24,7 @@ Ensure that you use markdown syntax for articles, we do not accept `HTML` or `CS
 
 **Commits and Squashing:**
 - Your Pull Request/Submission should contain one single commit.
-- For multiple commits follow the [guide to squash them](git-rebase#squashing-multiple-commits-into-one).
+- For multiple commits follow the [guide to squash them](Git-Squash).
```

Comment:
```diff
- [guide to squash them](Git-Squash)
+ [guide to squash them](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Git-Squash)
```

That relative URL won't work but this will `[guide to squash them](../Git-Squash)` but it might conflict with the camperbot regex. Please confirm @Rafase282